app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,25 @@
+@import "bootstrap";
+
+/* universal */
+
+body {
+  padding-top: 20px;
+}
+
+section {
+  overflow: auto;
+}
+
+.center {
+  text-align: center;
+  h1 {
+    margin-bottom: 10px;
+  }
+}
+
+ /* for calendar */
+
+.wrapper { display: -webkit-box; display: -ms-flexbox; display: flex; -webkit-box-orient: horizontal; -webkit-box-direction: normal; -ms-flex-direction: row; flex-direction: row; -webkit-box-pack: center; -ms-flex-pack: center; justify-content: center; -ms-flex-line-pack: center; align-content: center; }
```

Comment:
its better if you use prettify css for this code.

Reply:
done